internal/scheduler/scheduler_metrics.go

```diff
@@ -78,6 +78,23 @@ var actualSharePerQueueDesc = prometheus.NewDesc(
 	}, nil,
 )
 
+var demandPerQueueDesc = prometheus.NewDesc(
+	fmt.Sprintf("%s_%s_%s", NAMESPACE, SUBSYSTEM, "demand"),
+	"Demand of each queue and pool.",
+	[]string{
+		"queue",
+		"pool",
+	}, nil,
+)
+
+var fairnessErrorDesc = prometheus.NewDesc(
+	fmt.Sprintf("%s_%s_%s", NAMESPACE, SUBSYSTEM, "fairness_error"),
+	"Cumulative delta between adjusted fair share and actual share for all users who\n// are below their fair share",
+	[]string{
+		"pool",
```

Comment:
I was just curious why we don't have this metric per queue also, we could also sum them to get the total number